.github/workflows/release_production.yml

```diff
@@ -89,9 +89,10 @@ jobs:
         id: extract_version
         run: echo "RELEASE_VERSION=$(cat dist/version.txt)" >> $GITHUB_ENV
       - name: Send Slack Notification
-        uses: "./.github/actions/notify_slack"
+        env:
+          RELEASE_TYPE: Production
+          APP_NAME: smarttrader-deriv-app
+        uses: "deriv-com/shared-actions/.github/actions/send_slack_notification@master"
         with:
           SLACK_WEBHOOK_URL: ${{ secrets.SLACK_WEBHOOK_URL }}
-          status: ${{ env.WORKFLOW_CONCLUSION }}
-          release_type: Production
-          version: ${{ steps.extract_version.outputs.RELEASE_VERSION }}
+          MESSAGE: "$RELEASE_TYPE Docker Publish and Kubernetes Deployment for $APP_NAME with version ${{ needs.build_test_and_publish.outputs.RELEASE_VERSION }} has Failed"
```

Comment:
We should compose the slack message like here - https://github.com/binary-com/deriv-app/pull/14295/files#diff-93c7ed8fd6afff1de03995a214df3faa4dcd98168fed9759c

Can we check this?

<img width="1359" alt="image" src="https://github.com/deriv-com/smarttrader/assets/56330681/c8301ed0-a4b4-4a86-83db-c2814b96e567">
17e5db6e3147a09R77-R89